### DISCUSSION TITLE
Paint Bucket

### DISCUSSION THREAD

**Discussion by AhmetPolat43** [OpenOrienteering/mapper#1809] [General]
Will the Paint Bucket tool be attached to the android ? And I asked if it would be a problem for the mapper learning course? But didn't you answer?

**Comment by dg0yt** [comment]
> Will the Paint Bucket tool be attached to the android?

I assume you mean the "Fill bounded areas" tool. Probably it is not the only tool which is missing from the Android user interface. The screen is already quite cluttered. Adding more buttons won't relax this situation.

**Comment by AhmetPolat43** [comment]
I think one of the most important tools is that it does not need to be added, so it increases our drawing time in the field as you can see below
![7](https://user-images.githubusercontent.com/62382116/103193963-1adce580-48ef-11eb-9aaf-f46897ea4aef.jpg)
![8](https://user-images.githubusercontent.com/62382116/103193964-1ca6a900-48ef-11eb-9d8e-24a8c4e3c25c.jpg)
![00000000](https://user-images.githubusercontent.com/62382116/103193969-216b5d00-48ef-11eb-8dd8-2f4897ad39ce.png)

**Comment by AhmetPolat43** [comment]
We will fill the area directly, even on the phone.
